docs/guides/getting-started-java-spring.md

```diff
@@ -0,0 +1,255 @@
+
+## Step 1: Install Camunda 8 Self-Managed
+
+If you haven't already, ensure you have Camunda 8 Self-Managed installed locally:
+
+1. Use the `docker-compose.yaml` file in [this repository](https://github.com/camunda/camunda-platform).
+2. Clone this repo and run `docker compose up -d` in your terminal to start your environment.
+
+To confirm Camunda 8 Self-Managed is installed, click into Docker Desktop. Here, you will see the `camunda-platform` container. Alternatively, navigate to the different components and log in with the username `demo` and password `demo`. For example, Operate can be accessed at [http://localhost:8081](http://localhost:8081) (as noted under **Port(s)** in the Docker container). Find additional guidance in the repository [README](https://github.com/camunda/camunda-platform?tab=readme-ov-file#using-docker-compose).
+
+## Step 2: Create a new Spring Boot project
+
+Next, create a new Spring Boot project:
+
+1. Go to [https://start.spring.io/](https://start.spring.io/) to get started.
+2. Under **Project**, select **Maven**. Under **Language**, select **Java**. Under **Spring Boot**, select the latest non-SNAPSHOT version.
+3. Under **Project Metadata**, configure the following:
+   1. **Group**: `io.camunda.demo`
+   2. **Artifact**: `process_payments`
+   3. **Name**: `Process payments`
+   4. **Description**: `Process payments with Camunda`
+   5. **Package name**: `io.camunda.demo.process_payments`
+   6. **Packaging**: `Jar`
+   7. **Java**: Select the Java version you have installed.
+   8. For this tutorial, we will not install any dependencies.
+4. Click **Generate**.
+5. Download the project, click into the `.zip` file, and add it to your desired location.
+6. Open this project in your preferred code editor.
+7. Run `mvn spring-boot:run` in your terminal to confirm your Spring project builds.
+8. (Optional) Run `git init` if you'd like to commit milestones along the way, and add a `.gitignore` file with `target/` to ignore build artifacts.
+
+## Step 3: Create a new BPMN diagram
+
+Next, we'll create a BPMN diagram to represent the transaction model shown at the beginning of this guide:
+
+1. Open Desktop Modeler.
+2. Click **Create a new diagram** in Camunda 8, and name your diagram `Process payments` with an id of `process-payments`.
+3. Add a start event, and name it `Payment request received`.
+4. Add a task named `Prepare transaction`. Click the wrench-shaped icon to change the type of task to a script task, and configure the following properties:
+   1. **Implementation**: `FEEL expression`
+   2. **Script/Result variable**: `totalWithTax`
+   3. **Script/FEEL expression**: `total * 1.1` (this represents the tax applied to the transaction.)
+5. Add a task named `Charge credit card`.
+6. Click on the task and click the wrench-shaped icon to change the type of task to a service task. In the properties panel, change the **Task definition/Type** to `charge-credit-card`.
```

Comment:
Step 4 was "add a task" and "modify the task" combined into one step. Steps 5 & 6 have those steps separated. I don't have a preference between the two approaches, but I think we should be consistent.

Reply:
Resolved in latest commit 👍